lib/engine/config.ex

```diff
@@ -121,6 +124,29 @@ defmodule Engine.Config do
     {:noreply, Map.put(state, :current_version, latest_version)}
   end
 
+  def handle_info(:update_active_headend, state) do
+    schedule_update_active_headend(self())
+    updater = Application.get_env(:realtime_signs, :external_config_getter)
+
+    case updater.get_active_headend_ip() do
+      {:ok, active_headend_ip} ->
+        # Update active headend ip if startup or after at least two consecutive checks
```

Comment:
I don't understand why we need to wait for two consecutive results. Can you clarify?

Reply:
It's a suggestion by Andrew to account for potential false positives. He thinks they're unlikely but he suggested waiting for at least two in a row to be safe.

I'm not sure if it's really necessary and am fine with removing it. But it also doesn't really hurt to have the check imo. It does create some potential for delay in the cutover but we're able to handle that and it would be basically imperceptible from the rider perspective